auth_backends/__init__.py

```diff
@@ -3,4 +3,4 @@
  These package is designed to be used primarily with Open edX Django projects, but should be compatible with non-edX
  projects as well.
 """
-__version__ = '0.5.3'  # pragma: no cover
+__version__ = '0.6.0'  # pragma: no cover
```

Comment:
We have a habit of leaving our packages (and APIs) at major version zero after the public API has stabilized. According to semver, major version zero is only intended for initial development. We're using this package everywhere now, which suggests the initial development phase is over.